packages/fast-components-react-msft/src/subheading/README.md

```diff
@@ -0,0 +1,8 @@
+# Subheading
+Use various subheading sizes to create a clear hierarchy within your page. Subheadings are paired with HTML heading elements such as `<h1>` through `<h6>` and `<p>`.
+
+## Usage
+Use the appropriate numbered tag for the relative importance of each subheading compared to other subheadings on the page. Subheadings should be descriptive summaries of the content below them. Subheading cannot exist without a heading. Choose a subheading level that best organizes your page content. Then choose the subheading level to meet your design needs. For example, an `<h2>` element does not absolutely require level 2, it could be any of the other levels as well.

```

Comment:
Maybe remove "Subheading cannot exist without a heading. " from this paragraph - we don't really enforce that behavior in this library.